platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md

```diff
@@ -313,11 +313,21 @@ You bring:
 - Link to your [test plan](https://dsvavsp.testrail.io/index.php?/suites/view/2&group_by=cases:section_id&group_order=asc) in TestRail with [test results](https://dsvavsp.testrail.io/index.php?/runs/view/7&group_by=cases:section_id&group_order=asc) _for QA practice area review_ 
```

Comment:
Might we clarify this to `Link to your [QA test plan]`?

Reply:
👍 Done.